setup/kubernetes/k3s.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "k3s"
+description: Set up a Coder deployment with a Ubuntu + k3s
+---
+
+k3s is a lightweight Kubernetes distribution that works well for single-node or
+multi-node clusters alike.
+
+> This guide only covers installation on a new Ubuntu 20.04 LTS machine. If you
+> are looking to install on a local machine or an existing host,
+> [Kind](./kind.md) or k3d may be a better choice.
+
+## Prerequisites
+
+Before proceeding, please make sure that you have the following:
+
+1. **Ubuntu 20.04 machine**: This can be a bare metal or virtual machine. For
+   cloud compute, AWS EC2, GCP Compute Engine, Azure VMs, Vultr, DigitalOcean
+   work well. Make sure the machine specs satisfies Coder's
+   [resource requirements](../requirements.md)
+1. Network policy exposting ports 22, 80, 443, and 5522 (alternative SSH port)
```

Comment:
```suggestion
1. Network policy or firewall accepting incoming traffic on ports 22, 80, 443, and 5522 (alternative SSH port)
```

Reply:
You should only list the ports that coder requires here, so I wouldn't list 5522 (but make sure to note the firewall thing where you say the host ssh port needs to be changed)

Reply:
👍🏼